extension/data/deinflect.dat

```diff
@@ -123,6 +124,8 @@ imperative negative
 かせる	く	513	9
 がせる	ぐ	513	9
 かった	い	1152	14
+ありません	ない	1152	13
```

Comment:
I just thought that it would be nice if the user can see that じゃありません and ではありません comes from じゃない and ではない first and can also see that they are also inflections of だ.

Reply:
Actually, in that case can we make these more specific by including the じゃ and では in the mappings.

If I hover over ありません now it has some strange inflections for cases which don't make sense I think (adjectives which are just ない for example)

Reply:
(I updated this comment yesterday but forgot to unresolve it. sorry!)

Reply:
Ah, I see. I will fix it immediately